Makefile

```diff
@@ -109,7 +132,10 @@ clean: tidy
 
 tidy:
 	rm -f $(ROM) $(ELF) $(MAP)
+ifeq ($(MODERN),0)
+	@$(MAKE) tidy MODERN=1
 	rm -r build/*
```

Comment:
I think this should be `rm -r $(OBJ_DIR)`?

Reply:
Fixed